tests/dragonfly/replication_test.py

```diff
@@ -49,14 +49,16 @@ async def wait_for_replicas_state(*clients, state="online", node_role="slave", t
         # Quick general test that replication is working
         (1, 3 * [1], dict(key_target=1_000), 500),
         # A lot of huge values
-        (2, 2 * [1], dict(key_target=1_000, huge_value_percentage=2), 500),
+        (2, 2 * [1], dict(key_target=1_000, huge_value_target=30), 500),
```

Comment:
same - does it belong to this PR?

Reply:
I changed the logic of the seeder for huge values
Now we have by default huge values in seeder, which are the once who makes us trigger the big value serialization flow.
The problem with huge_value_percentage is that if we have large key_target this will lead also to large target of huge values and we can get to OOM fast in this case.
Before the flow was enabled because the value of serialization_max_chunk_size was kind of small leading us to test the big value serialization flow but it caused the regular flow of SliceSnapshot::PushSerialized to be disabled as we always flush before.